sidebar/Instructions.txt

```diff
@@ -0,0 +1,13 @@
+Hey, so I've tried to create a sidebar menu that will display the housing stock for municipalities of Groningen and Drenthe.
+I found it as a very tidious task, and it is still in progress.
+I think the way I added this input is very messy, I def need to improve the way I add my input, 
+hence this instructions/remarks.
+I don't know if you need it, but in case you face issues to recreate what I did, here it is.
+
+1) I still did not figure out how to read data from json, may be you can help me with that. So I read it from a csv.
+For now, I uploaded the data set on the git in the folder datasets, both in .csv and .json.
+
+2) Run housing_stock_data_municipalities.R with the dataset I uploaded.
+3) Run Clean_plot.R
+4) replace main_ui_withsidebar with the original main_ui
+5) Run shiny 
```

Comment:
This is very informative, thanks! Please put it in your commit message or in your pull request though. Having files like this laying around in our codebase makes it a mess.